src/core_ocean/Registry.xml

```diff
@@ -1647,7 +1626,8 @@
 				mode="forward;analysis">
 
 			<stream name="mesh"/>
-			<var_struct name="tracers"/>
+      <var_struct name="tracers"/>
+      <var name="indMLD"/>
```

Comment:
@vanroekel I'm taking a last careful look through. We added `indMLD` to the restart file here when we were struggling with the restart bug, but I don't think it is needed. The restart test passes without it, with Redi and GM on. Do you agree it can be removed?

Reply:
I think so.  I believe rediGM is now called before the timestep but after the analysis compute, but this might require that MLD be computed on start up.  Is that flag set in your testing?  If we don't compute on start up we may want to keep indMLD in the restart file.

Reply:
I guess it depends on the timing of the `mixedLayerDepths` AM. We can only safely remove the variable if the AM is computed just before the restart is written, to get exact restarts. For example, if we use either of these settings:
```
config_AM_mixedLayerDepths_compute_interval = 'dt'
config_AM_mixedLayerDepths_compute_interval = '01_00:00:00'
```
for a global run, we can remove `indMLD` from the restart file. The safest option is to leave it in, at the cost of a slightly larger restart file.